project/profiles/capacity4more/modules/c4m/content/c4m_content/c4m_content.module

```diff
@@ -67,6 +67,23 @@ function c4m_content_form_node_form_alter(&$form, &$form_state, $form_id) {
 }
 
 /**
+ * Implements hook_node_submit().
+ *
+ * If 'Send Notification' checkbox exists, add a parameter with value to node.
```

Comment:
`... to the node in order to determine if we should send the notification on the hook_node_update / insert`